documentation/staging/content/faq/node-heating.md

```diff
@@ -29,12 +29,37 @@ clusters:
                     operator: In
                     values:
                     - $(CLUSTER_NAME)
+                  - key: "weblogic.domainUID"
+                    operator: In
+                    values:
+                    - $(DOMAIN_UID)
               topologyKey: "kubernetes.io/hostname"
 ```
 
-Because the `serverPod` element here is scoped to a cluster, the content of the `affinity` element will be added to the Pod generated for each WebLogic Server instance that is a member of this WebLogic cluster. This inter-pod anti-affinity statement expresses a preference that the scheduler select a Node for the new Pod avoiding, as much as possible, Nodes that already have Pods with the label "weblogic.clusterName" and the name of this cluster. Note that the `weight` is set to `100`, which is the maximum weight, so that this term will outweigh any possible preference for a Node based on availability of container images.
+Because the `serverPod` element here is scoped to a cluster, the content of the `affinity` element will be added to the Pod generated for each WebLogic Server instance that is a member of this WebLogic cluster. This inter-pod anti-affinity statement expresses a preference that the scheduler select a Node for the new Pod avoiding, as much as possible, Nodes that already have Pods with the label "weblogic.clusterName" and the name of this cluster and with the label "weblogic.domainUID" and the UID of this domain. Note that the `weight` is set to `100`, which is the maximum weight, so that this term will outweigh any possible preference for a Node based on availability of container images.
```

Comment:
"weblogic.clusterName" -> `weblogic.clusterName` (Instances where you have double quotes, should be code font, no quotes. I know it was there previously but let's fix it now)
"weblogic.clusterName" and the name of this cluster -> `weblogic.clusterName` and the name of this cluster, (comma)
"weblogic.domainUID" -> `weblogic.domainUID`

Reply:
Done.